gulpfile.js

```diff
@@ -80,25 +80,33 @@ gulp.task('test.e2e', ['test.compile'], function(done) {
 
```

Comment:
Yes, this needs to evolve further. I also think we should pull the unittest dep into a separate file (maybe code-gen'd) so we can also execute the tests in TypeScript.